code/modules/shuttle/gamemodes/crash.dm

```diff
@@ -40,8 +40,13 @@
 	var/turf/rear_left = locate(C.x + round(leftright/2), C.y + round(updown/2), C.z)
 
 
+	// just wipe the turfs
+	for(var/turf/T in range(3, rear)+range(3, left)+range(3, right)+range(3, front)+range(4, front_right)+range(4, front_left)+range(4, rear_right)+range(4, rear_left))
+		T.empty(/turf/open/floor/plating)
+
 	// Big explosions
-	explosion(front, 3, 4, 7, 0)
+
+/*	explosion(front, 3, 4, 7, 0)
```

Comment:
Replace the explosions with a smoke effect?
Would be harmless, but still give the same type of effect

Reply:
sure but it'd need to be after landing not before

Reply:
That'd be fine

Reply:
outside the scope of this pr :P